jobs/cronjob-ldap-group-sync.yml

```diff
@@ -7,44 +7,64 @@ metadata:
     description: "Scheduled Task to Perform LDAP Group Synchronization"
     iconClass: "icon-shadowman"
     tags: "management,cronjob,ldap,group,sync"
+labels:
```

Comment:
Will labels at this level apply to all objects within this template?

Reply:
Yes. Double checked just now.